### PR TITLE
drivers: entropy: nrf5: Clarify Kconfig options

### PR DESCRIPTION
**Pull request by carlescufi**
Clarify the difference between the buffer length and threshold Kconfig
options available in the nRF5 entropy driver.

Signed-off-by: Carles Cufi <carles.cufi@nordicsemi.no>

### CODE REVIEW

File: drivers/entropy/Kconfig.nrf5 [hunk 1]
```diff
@@ -29,28 +29,30 @@ config ENTROPY_NRF5_THR_THRESHOLD
 	range 4 255
 	help
 	  Low-water threshold in bytes to trigger entropy generation for
-	  thread mode consumers.
+	  thread mode consumers. As soon as the currently available bytes go
+	  below this mark hardware entropy generation will be started.
```
Comment (IvanSanchez) [inline drivers/entropy/Kconfig.nrf5]: Nitpicking: "As soon as there are less available bytes than this threshold, hardware entropy generation will start"

File: drivers/entropy/Kconfig.nrf5 [hunk 2]
```diff
@@ -29,28 +29,30 @@ config ENTROPY_NRF5_THR_THRESHOLD
...
 	range 12 255
 	help
 	  Low-water threshold in bytes to trigger entropy generation for
-	  ISR consumers.
+	  ISR consumers. As soon as the currently available bytes go
+	  below this mark hardware entropy generation will be started.
 
 config ENTROPY_NRF5_THR_BUF_LEN
 	int "Thread-mode buffer length"
 	range ENTROPY_NRF5_THR_THRESHOLD 255
```
Comment (IvanSanchez) [inline drivers/entropy/Kconfig.nrf5]: Is this the max number of entropy bytes as well?

**Review by IvanSanchez — APPROVED** [review]

File: drivers/entropy/Kconfig.nrf5 (hunk 2 above)
Comment (carlescufi) [reply drivers/entropy/Kconfig.nrf5]: No, the buffer length is the max number of bytes you can retrieve from ISR in one call

File: drivers/entropy/Kconfig.nrf5 (hunk 1 above)
Comment (carlescufi) [reply drivers/entropy/Kconfig.nrf5]: ok will rephrase a bit

**Review by SebastianBoe — APPROVED** [review]

**Review by nashif — APPROVED** [review]

**Pull request merged**